front/src/components/notification/notificationsFrequency.component.test.tsx

```diff
@@ -0,0 +1,69 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import type { RenderAPI } from "@testing-library/react-native";
+import { render, waitFor } from "@testing-library/react-native";
+import * as React from "react";
+
+import { StorageKeysConstants } from "../../constants";
+import { NotificationUtils, StorageUtils } from "../../utils";
+import { NotificationType } from "../../utils/notifications/notification.util";
+import NotificationsFrequency from "./notificationsFrequency.component";
+
+describe("Notification Frenquency", () => {
+  describe("UI", () => {
+    // eslint-disable-next-line @typescript-eslint/init-declarations
+    let screen: RenderAPI;
+
+    afterEach(() => {
+      void AsyncStorage.clear();
+    });
+
+    it("show for the first time", async () => {
```

Comment:
est-ce qu'on pourrait renommer avec par exemple `moodboard frequency init state should be twice a week`

et idem pour le screen, c'est le même dans tes deux tests, est-ce qu'on pourrait l'init qu'une seule fois ? :)

Reply:
idem, il y a du localstorage avant dans certains tests